verticaltabs.jsm

```diff
@@ -438,6 +463,7 @@ VerticalTabs.prototype = {
       tabs.removeEventListener('TabPinned', this, false);
       tabs.removeEventListener('TabUnpinned', this, false);
       tabs.removeAttribute('vertical');
+      tabs.removeChild(document.getElementById('filler-tab'));
```

Comment:
Let's try removing this code, and seeing if the element disappears automatically, then.  😃